app/views/cases/sar/_case_details.html.slim

```diff
@@ -0,0 +1,22 @@
+.grid-row
+  .column-full.case-details
+    h2.request--heading
+      = "Case details"
+
+    table
+      tbody.sar-basic-details
+        = render partial: 'cases/shared/case_type', locals: { case_details: case_details }
+        = render partial: 'cases/shared/sar_data_subject', locals: { case_details: case_details }
```

Comment:
Why not keep these in the `sar` sub-folder instead if filling up `shared` with type-specific partials?